cfme/common/__init__.py

```diff
@@ -191,9 +191,11 @@ def is_displayed(self):
 class WidgetasticTaggable(object):
     """
     This class can be inherited by any class that honors tagging.
-    Class should have 'EditTags' navigate step registered,
-    also 'Details' view should be present with tag attribute(widget with tag
-    information on details page)
+    Class should have following:
+        - 'Details' navigation
+        - 'Details' view should have entities.smart_management SummaryTable widget
+        - 'EditTags' navigation
+        - 'EditTags' view should have nested view.form for add and view.tags table widget for remove
```

Comment:
it is not mandatory to have 'EditTags' view with a form, we can use common TagPageView in EditTags navigation step

Reply:
I see this class now - there's one in common/vm_views as well. I'll revise this a bit, but I'm thinking the common view should have a nested form as we've done that on other pages. the title/buttons are not in the nested form, but in the top-level view. Only form widgets are in the nested 'form' view.